collective/googlenews/tests/test_digitid.py

```diff
@@ -1,27 +1,19 @@
 # -*- coding: utf-8 -*-
-
+from collective.googlenews import digitid
 from collective.googlenews.tests import base, utils
 
 
 class Test(base.UnitTestCase):
 
-    def setUp(self):
-        super(Test, self).setUp()
-#        from collective.gallery import core
-#        self.view = core.BaseBrowserView(self.context, self.request)
-
     def test_randomid(self):
-        from collective.googlenews import digitid
         id = digitid.randomid()
         # check common rule
         self.failUnless(id.startswith('-'))
-        self.failUnless(id.endswith('.html'))
         # check on with 1000 ids
-        digits = [digitid.randomid()[1:-5] for i in range(1000)]
+        digits = [digitid.randomid()[1:] for i in range(1000)]
         for digit in digits:
-            self.failUnless(digit.isdigit())
-            self.failUnless(len(digit) > 2)
-            self.failUnless(len(digit) < 5)
+            self.assertTrue(digit.isdigit())
+            self.assertTrue(100 <= int(digit) <= 9999)
```

Comment:
I always think it is good to provide the base when convert to int to avoid problems when the number starts with 0 and python think it is an octal..
So I prefer if change to `int(digit, 10)`